### PR TITLE
swhid_compute.0.1

### PR DESCRIPTION
**Pull request by zapashcanon**

### CODE REVIEW

File: packages/swhid_compute/swhid_compute.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ocamlpro/swhid_compute.git"
+url {
+  src: "https://github.com/OCamlPro/swhid_types/archive/refs/tags/0.1.tar.gz"
```
Comment (kit-ty-kate) [inline packages/swhid_compute/swhid_compute.0.1/opam:46]:
```suggestion
  src: "https://github.com/OCamlPro/swhid_compute/archive/refs/tags/0.1.tar.gz"
```

Comment (zapashcanon) [reply packages/swhid_compute/swhid_compute.0.1/opam]: Thanks !

**Comment by kit-ty-kate** [comment]
Any chance you could add some `(enabled_if %{archsixtyfour})` where needed to fix those issues?
```
#=== ERROR while compiling swhid_compute.0.1 ==================================#
# context              2.1.2 | linux/x86_32 | ocaml-base-compiler.4.14.0 | pinned(https://github.com/OCamlPro/swhid_compute/archive/refs/tags/0.1.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/swhid_compute.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p swhid_compute -j 31 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/swhid_compute-2581-afe55d.env
# output-file          ~/.opam/log/swhid_compute-2581-afe55d.out
### output ###
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -g -bin-annot -I test/.test.eobjs/byte -I /home/opam/.opam/4.14/lib/astring -I /home/opam/.opam/4.14/lib/bigarray-compat -I /home/opam/.opam/4.14/lib/bos -I /home/opam/.opam/4.14/lib/digestif/c -I /home/opam/.opam/4.14/lib/eqaf -I /home/opam/.opam/4.14/lib/fmt -I /home/opam/.opam/4.14/lib/fpath -I /home/opam/.opam/4.14/lib/logs -I /home/opam/.opam/4.14/lib/rresult -I /home/opam/.opam/4.14/lib/stdlib-shims -I /home/opam/.opam/4.14/lib/swhid_types -I src/.swhid_compute.objs/byte -no-alias-deps -o test/.test.eobjs/byte/dune__exe__Test.cmo -c -impl test/test.ml)
# File "test/test.ml", line 102, characters 39-49:
# 102 |            { Swhid_compute.timestamp = 1130457753
#                                              ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
```

**Comment by zapashcanon** [comment]
I went with `Int64.t` instead. Thanks ! :)

**Pull request merged**